de.dentrassi.pm.aspect.mvnosgi/src/de/dentrassi/pm/aspect/mvnosgi/internal/Constants.java

```diff
@@ -0,0 +1,10 @@
+package de.dentrassi.pm.aspect.mvnosgi.internal;
+
+public class Constants
+{
+    public static final String POM_FILE_NAME = "mvnosgi.pom";
```

Comment:
But I already complained it on your last change. ;-)
https://github.com/ctron/package-drone/issues/71#issuecomment-95068819